sections/05_treat.md

```diff
@@ -78,6 +78,53 @@ able to significantly outperform a random forest classifier trained on gene
 expression changes. These approaches suggest promising directions to improve
 the efficiency of clinical trials and accelerate drug development.
 
+##### Drug repositioning
+
+Drug repositioning (or repurposing) is an attractive option for delivering new
+drugs to the market because of the high costs and failure rates associated with
+more traditional drug discovery approaches [@doi:10.1016/j.jhealeco.2016.01.012
+@doi:10.1038/nrd4609]. Ever since the concept of the Connectivity Map was
+proposed [@doi:10.1126/science.1132939], several advanced computational methods
+have been applied to formulate and validate drug repositioning hypotheses
+[@doi:10.1093/bib/bbv020 @doi:10.1093/bib/bbw112 @doi:10.1093/bib/bbw110]. Using
```

Comment:
Do you think we need to point out that most of the methods covered in these reviews are not using deep learning?  It may not be necessary.